README.md

```diff
@@ -8,7 +8,7 @@ WARNING: cusdeb-single-node depends on some code which is not yet published in G
 
 Install packages.
 ```bash
-sudo apt install git docker docker-compose curl pkg-config pandoc supervisor python3-pip virtualenv postgresql postgresql-server-dev-all qemu-user-static whois -y
+sudo apt install git docker docker-compose curl pkg-config pandoc supervisor python-dev bison flex swig nodejs npm python3-pip virtualenv postgresql postgresql-server-dev-all qemu-user-static whois -y
```

Comment:
@leocoan can you remember which node version it required?
Would be great if we can provide an instruction how to get the exactly version it requires

Reply:
We need Node.js 8 but it's not necessary to have on your machine. If you don't have it on your machine, single-node will install Node.js for you locally.